packages/desktop-client/src/components/spreadsheet/index.ts

```diff
@@ -89,15 +89,24 @@ export type SheetNames = keyof Spreadsheets & string;
 export type SheetFields<SheetName extends SheetNames> =
   keyof Spreadsheets[SheetName] & string;
 
+export type BindingObject<
+  SheetName extends SheetNames,
+  SheetFieldName extends SheetFields<SheetName>,
+> = {
+  name: SheetFieldName;
+  value?: Spreadsheets[SheetName][SheetFieldName] | undefined;
+  query?: Query | undefined;
+};
+
 export type Binding<
   SheetName extends SheetNames,
   SheetFieldName extends SheetFields<SheetName>,
 > =
   | SheetFieldName
   | {
```

Comment:
This should be same as `BindingObject` but TypeScript complains when I union it with `BindingObject` vs. delaring it as an object.